components/layout/ActivityList.js

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+import ActivityCard from "@/components/layout/ActivityCard";
+import ActivityForm from "@/components/layout/ActivityCreateForm";
```

Comment:
Please don't rename your component on the fly. I recommend to rename the file to 'ActivityForm'.